### PR TITLE
Integrates cloud fixity statuses into the fixity dashboard display

### LINKED ISSUE #2910: Fixity Dashboard should show all failures and recent successes of Cloud Fixity operations

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Resolves #2910 by addressing the following:
- Implements the `FindCloudFixityFailures` and `FindCloudFixityChecked` custom queries
- Integrates cloud fixity statuses into the dashboard display

### CODE REVIEW

File: app/queries/find_cloud_fixity_checked.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  delegate :orm_class, to: :resource_factory
+  def initialize(query_service:)
+    @query_service = query_service
+  end
+
+  def query(order_by_property: "updated_at", order_by: "ASC")
+    <<-SQL
+      SELECT b.* FROM
+        (SELECT json_array_elements(json_array_elements(json_extract_path(a.metadata::json,'file_metadata'))->'file_identifiers') AS file_identifier, a.* FROM orm_resources AS a) AS b
+        WHERE b.metadata @> ? AND b.file_identifier->>'id' LIKE 'https://storage.googleapis.com/%' ORDER BY b.#{order_by_property} #{order_by} LIMIT ?
```
Comment (jrgriffiniii) [inline app/queries/find_cloud_fixity_checked.rb]: This should be modified to check for `shrine://` identifiers: https://github.com/samvera-labs/valkyrie-shrine/blob/master/lib/valkyrie/storage/shrine.rb#L9

**Comment by jrgriffiniii** [comment]
This needs to query for `Event` resources with successful `status` values, not query for `FileSets`.

File: app/controllers/fixity_dashboard_controller.rb [hunk 2]
```diff
@@ -3,6 +3,9 @@ class FixityDashboardController < ApplicationController
   delegate :query_service, to: :metadata_adapter
 
   def show
+    @cloud_failures = query_service.custom_queries.find_by_property(property: :status, value: "FAILURE").map(&:decorate)
```
Comment (tpendragon) [inline app/controllers/fixity_dashboard_controller.rb]: We don't have to change this now, but we should keep an eye on the performance of this. I wouldn't be surprised to find we can improve this in the future by adding some way to filter by model.

File: app/queries/find_cloud_fixity_checked.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  def query(order_by_property: "updated_at", order_by: "ASC")
+    <<-SQL
+      select * FROM orm_resources WHERE
+      metadata @> ?
+      ORDER BY #{order_by_property} #{order_by} LIMIT ?
+    SQL
+  end
+
+  def find_cloud_fixity_checked(sort: "ASC", limit: 50, order_by_property: "updated_at")
+    internal_array = { "status" => Array.wrap("SUCCESS") }
```
Comment (tpendragon) [inline app/queries/find_cloud_fixity_checked.rb]: Instead of `Array.wrap` can you just do `["SUCCESS"]`?

**Pull request merged**